boot/baselib/base.scm

```diff
@@ -37,6 +37,11 @@
                (let ((it ,sym)) ,@(cdr cl1))
                (acond ,@(cdr clauses)))))))
 
+;; R7RS features.
+;; TODO(higepon): Add more.
+(define (available-features)
+  '(r6rs r7rs mosh mosh-0.2.8))
```

Comment:
We shouldn't expose `mosh-0.2.8` since we have not reached any stable state.

Reply:
Thanks for the catch.